extensions/crossmodel-lang/src/glsp-server/model/builders/GEntityNode.ts

```diff
@@ -0,0 +1,75 @@
+/********************************************************************************
+ * Copyright (c) 2023 CrossBreeze.
+ ********************************************************************************/
+import { ArgsUtil, GCompartment, GLabel, GNode, GNodeBuilder } from '@eclipse-glsp/server';
+import { DiagramNode } from '../../../language-server/generated/ast';
+
+export class GEntityNode extends GNode {
+   override type = 'node:entity';
+
+   static override builder(): GEntityNodeBuilder {
+      return new GEntityNodeBuilder(GEntityNode).type('node:entity');
+   }
+}
+
+export class GEntityNodeBuilder extends GNodeBuilder {
+   addNode(node: DiagramNode): this {
+      // Get the reference that the DiagramNode holds to the Entity in the .langium file.
+      const entityRef = node.semanticElement.ref;
+
+      // Options which are the same for every node
+      this.addCssClasses('diagram-node', 'entity').layout('vbox').addArgs(ArgsUtil.cornerRadius(3));
+
+      // We need the id before we can build the label and childeren.
+      if (this.id === undefined) {
+         throw new Error('Add id to builder before adding the node reference.');
+      }
+
+      // Add the label/name of the node
+      const label = GCompartment.builder()
+         .layout('hbox')
+         .addLayoutOption('hAlign', 'center')
+         .addCssClass('header-container-diagram-node')
```

Comment:
I think heder-compartment would be a better css class name (just like with attributes.

Reply:
done